demo/MINDCUB3R/README.md

```diff
@@ -1,26 +1,13 @@
 # MINDCUB3R
 
 ### Installation
-This can take a minute on an EV3
-* cd ~/ev3dev-lang-python/demo/MINDCUB3R/kociemba/ckociemba/
-* sudo apt-get update
-* sudo apt-get install build-essential
-* make
-* sudo make install
-
-The first time you run this it creates a cache directory, this takes about 30s on an EV3
-
-* cd ~/ev3dev-lang-python/demo/MINDCUB3R/
-* kociemba DRLUUBFBRBLURRLRUBLRDDFDLFUFUFFDBRDUBRUFLLFDDBFLUBLRBD
-
+This can take several minutes on an EV3
+```
+sudo apt-get install build-essential python3-pip libffi-dev
+sudo pip3 install git+https://github.com/dwalton76/kociemba.git
+sudo pip3 install git+https://github.com/dwalton76/rubiks-color-resolver.git
+kociemba DRLUUBFBRBLURRLRUBLRDDFDLFUFUFFDBRDUBRUFLLFDDBFLUBLRBD
```

Comment:
I have that step in the install process because the first time you run it, it takes about 30 seconds because it builds some tables that it caches to the filesystem.  After that first run it is nice and fast.